.release-it.json

```diff
@@ -1,10 +1,15 @@
 {
   "requireUpstream": false,
   "increment": "conventional:angular",
-  "buildCommand": "conventional-changelog -p angular -i CHANGELOG.md -s -r 0 && git add CHANGELOG.md",
+  "buildCommand": "npm run changelog && git add CHANGELOG.md",
   "src": {
     "commitMessage": "chore(release): %s",
     "beforeStartCommand": "npm run lint && npm run test -- --colors && npm run test:e2e -- --colors",
     "afterReleaseCommand": "npm run release-templates"
+  },
+  "github": {
+    "release": true,
+    "releaseName": "%s",
+    "tokenRef": "GITHUB_TOKEN_CISA"
```

Comment:
where does the token come from?

Reply:
I explain it in the [contributing guide](https://github.com/algolia/create-instantsearch-app/pull/61/files#diff-6a3371457528722a734f3c51d9238c13R69). Does that make sense?